tensorflow/stream_executor/rocm/BUILD

```diff
@@ -5,9 +5,11 @@ load(
     "//tensorflow/stream_executor:build_defs.bzl",
     "stream_executor_friends",
 )
+
 load("//tensorflow:tensorflow.bzl", "tf_copts")
-load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm_is_configured")
+load("@local_config_rocm//rocm:build_defs.bzl", "if_rocm_is_configured", "rocm_default_copts")
 load("//tensorflow/core/platform:build_config_root.bzl", "if_static")
+load("@local_config_cuda//cuda:build_defs.bzl","cuda_library")
```

Comment:
see if you can use the `tf_gpu_kernel_library` wrapper from `tensorflow.bzl` instead.

would really like to avoid references into cuda config from the rocm dir :)

Reply:
Tried that but bazel won't let me, because it creates a circular reference (tf_gpu_kernel_library has a dependence on stream_executor)